src/worker/fetch-events.js

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env node
+const _ = require('lodash');
+const google = require('googleapis');
+const util = require('../util/seeds');
+const logger = require('../util/logger')(__filename);
+const {knex} = require('../util/database').connect();
```

Comment:
I meant that destructuring is not available in node 4. If I remember correctly we run specifically node v4 in heroku, but I might be mistaken. The fix would be to not use destructuring here or use babel-node